packages/manager/apps/pci-rancher/src/_mock_/pciProject.ts

```diff
@@ -1,4 +1,4 @@
-import { PciProject, PciProjectPlanCode } from '@/api/api.type';
+import { PciProject, PciProjectPlanCode } from "@/api/api.type";
```

Comment:
```suggestion
import { PciProject, PciProjectPlanCode } from '@/api';
```

You could export the types from the api index